### PR TITLE
[documentation]: Update README with Java 11 as a setup requirement

### LINKED ISSUE #48: Update Readme with Java 11 Setup Requirements
Currently, if you clone the project without having Java 11 setup, you'll face gradle errors. Update the readme with these instructions

### PR DESCRIPTION
**Pull request by Mogakamo**
# Scope

I am updating the README.md to add Java 11 as a setup requirement. 

- [ ] I have followed the coding conventions
- [ ] I have added/updated necessary tests
- [ ] I have tested the changes added on a physical device
- [ ] I have run `./codeAnalysis.sh` to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
closes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,14 @@ and is optimized for phones and tablets of all shapes and sizes.
...
 ## Dependencies
 
-1. Jetpack Compose
-2. Coroutines - For Concurrency and Asynchronous tasks
-3. Ktor - For network requests
-4. Hilt - For Dependency Injection
-5. Crashlytics
-6. Coil - For Image Loading and Caching
-7. Lint Checks - [Ktlint](https://ktlint.github.io/)
+1. JAVA 11 - [Install](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html)
```
Comment (chepsi) [inline README.md:10]: I wouldn't say it is a dependency per sè. I would add it under `Running the Project` and then add steps, for different environments(Mac, Windows, Linux) to check if Gradle is using java 11

Comment (Mogakamo) [reply README.md:10]: ok. Let me adjust the changes

File: README.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ Android app for the 4th Android Developer conference- droidcon to be held in Nai
 This project is the Android app for the conference. The app supports devices running Android 5.0+,
 and is optimized for phones and tablets of all shapes and sizes.
 
+## Running the Project
+
+To ensure the project runs on your local environment ensure to you have Java 11 on your pc or if you don't have you can install it from [here](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). 
+
+If you have multiple installations of Java make sure to set Java 11 as your preferred version to run the project.
```
Comment (chepsi) [inline README.md:12]: Can we add steps to check if you've selected the right java version?
https://developer.android.com/studio/intro/studio-config#jdk

Comment (Mogakamo) [reply README.md:12]: Ok let me add it too

**Review by chepsi — APPROVED** [review]

**Review by wangerekaharun — APPROVED** [review]

**Comment by chepsi** [comment]
@Mogakamo very minor change requested. LGTM.

**Pull request merged**